packages/passport/README.md

```diff
@@ -39,8 +39,9 @@ yarn dev
 # Context: passport-sdk-sample-app in ./sdk-sample-app
 yarn dev
 
-# Alternatively, you can run both commands in one terminal via the below
-yarn workspace @imtbl/passport-sdk-sample-app dev & yarn workspace @imtbl/passport dev
+# Alternatively, you can run both commands in one terminal via the below concurrent yarn script
+# Context: passport-sdk-sample-app in ./sdk-sample-app
+yarn dev-with-sdk
```

Comment:
Can we move this to the sample app README please?

Reply:
Oh disregard me, didn't realise we'd previously merged both README's into a root README.